src/test/java/org/isf/medicalsinventory/Tests.java

```diff
@@ -938,6 +940,17 @@ void testValidateMedicalWardInventoryRow() {
 		});
 		// Test if exception is OHDataValidationException instance
 		assertThat(throwable).isInstanceOf(OHDataValidationException.class);
+		//  Debugging code
```

Comment:
```suggestion
		//  FIXME: Debugging code
```

Reply:
@ArnaudFofou I think we should come back to this when we have time... created OP-1392